ui-spacetimechart/src/components/workSchedules/WorkSchedules.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+
+import SpaceTimeRectangle from './SpaceTimeRectangle';
+import { type WorkSchedule } from '../../types';
+
+type WorkSchedulesProps = {
+  workSchedules: WorkSchedule[];
+  imageUrl: string;
+};
+
+/**
+ * Displays the workschedule projection on the Space time chart.
+ * A workschedule has a start time and an end time.
+ * It can contain several portions of a track occupied during this period.
+ * Each portion is represented by a rectangle region
+ */
+const WorkSchedules = ({ workSchedules, imageUrl }: WorkSchedulesProps) => {
```

Comment:
It's a suggestion, to harmonize the layers name
```suggestion
const WorkScheduleLayer = ({ workSchedules, imageUrl }: WorkSchedulesProps) => {
```

Reply:
done